.github/workflows/build-test.yml

```diff
@@ -106,6 +106,7 @@ jobs:
           drivers/test-fw/build.sh
           (cd fmc && ./build.sh)
           (cd runtime && ./build.sh)
+          (cd rom/dev && ./build.sh)
```

Comment:
Should we integrate the rom test as well?

Reply:
We will integrate them once we have them ready. Vishal Mhatre is working on that. Will be part of subsequent check ins. For now if we can compile is a good start.